libff/algebra/curves/tests/test_bilinearity.cpp

```diff
@@ -35,7 +38,8 @@ template<typename ppT> void pairing_test()
     Q.print_coordinates();
     printf("\n\n");
 
-    Fr<ppT> s = Fr<ppT>::random_element();
+    // Fr<ppT> s = Fr<ppT>::random_element(); // VV
+    Fr<ppT> s = Fr<ppT>("2");
```

Comment:
Likewise, temporary change?

Reply:
Removed.

Reply:
I still see the change. Looks like you changed:
`Fr<ppT> s = Fr<ppT>::random_element();` to `Fr<ppT> s = Fr<ppT>("2");` in this PR